src/rules/forbid-throw-new-error.ts

```diff
@@ -0,0 +1,37 @@
+import { createRule } from '../utils'
+
+const NAME = 'forbid-throw-new-error'
+
+const value = createRule({
+    create(context) {
+        return {
+            Identifier(node) {
+                if (node.name === "Error") {
```

Comment:
This completely forbids use of 'Error'?

Reply:
Yup, pretty much a shotgun approach. Will test it on the backend before making it official ofc.